github/ci/prow-deploy/files/jobs/kubevirt/terraform-provider-kubevirt/terraform-provider-kubevirt-presubmits.yaml

```diff
@@ -0,0 +1,76 @@
+presubmits:
+  kubevirt/terraform-provider-kubevirt:
+  - name: pull-terraform-proider-kubevirt-e2e-k8s-1.20
+    skip_branches:
+      - release-\d+\.\d+
+    annotations:
+      fork-per-release: "true"
+      testgrid-dashboards: kubevirt-presubmits
+    always_run: true
+    optional: false
+    skip_report: false
+    cluster: prow-workloads
+    decorate: true
+    decoration_config:
+      timeout: 7h
+      grace_period: 5m
+    max_concurrency: 11
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "false"
```

Comment:
nit: IIRC all [presets](https://github.com/kubernetes/test-infra/blob/master/config/jobs/README.md#job-presets) that have `"false"` as value can also be omitted. @fgimenez please keep me honest here. I figure we have a lot of copy paste there.

Reply:
Changed "preset-docker-mirror-proxy" to true (in both e2e-tests and unit-tests)